app/apptesting/test_helpers.go

```diff
@@ -231,7 +231,10 @@ func (s *AppTestHelper) CreateICAChannel(owner string) (channelID, portID string
 	_, transferChannelExists := s.App.IBCKeeper.ChannelKeeper.GetChannel(s.Ctx, ibctesting.TransferPort, ibctesting.FirstChannelID)
 	if !transferChannelExists {
 		ownerSplit := strings.Split(owner, ".")
-		s.Require().Equal(2, len(ownerSplit), "owner should be of the form: {HostZone}.{AccountName}")
+		isHostZoneICA := len(ownerSplit) == 2
+		isTradeRouteICA := len(ownerSplit) == 3
+		s.Require().True(isHostZoneICA || isTradeRouteICA,
+			"owner should be either of the form: {chainId}.{AccountName} or {chainId}.{rewarDenom}-{hostDenom}.{accountName}")
```

Comment:
hmm, at first I thought the tradeRoute one seems redundant but chainId would be the tradeZone chain Id right, so then it could be independent of the rewardDenom and hostDenom so all of that information really is needed.  My second thought was that for consistency we could use the chainIds for the reward Zone and hostZone but then I realized this is for convenience of being able to look up the TradeRoute from keeper.  So everything makes sense and is in fact needed even though it seemed like it should be simplified at first glance